thunder/dynamo/compiler.py

```diff
@@ -13,14 +31,15 @@ def _warn_thunder_compiler():
 
 
 class ThunderCompiler:
-    def __init__(self, **thunder_options):
+    def __init__(self, *, thunder_options: dict | None = None, torch_inductor_options: dict | None = None):
```

Comment:
I think it's less cognitive load to switch from `thunder.jit` to `ThunderCompiler` if the way to pass Thunder options is the same.
I suggest keeping `**kwargs` and then popping "torch_inductor_options" from this dict. For example specifying executors for Thunder and a torch.compile option would look like:
```py
backend = ThunderCompiler(executors=[...], torch_inductor_options={"mode": "max-autotune"})
compiled_module = torch.compile(backend=backend)(module)